pkgs/applications/networking/mailreaders/imapfilter.nix

```diff
@@ -1,27 +1,27 @@
-{ stdenv, fetchFromGitHub, openssl, lua, pcre }:
+{ stdenv, fetchFromGitHub, openssl, lua, pcre2 }:
 
 stdenv.mkDerivation rec {
   pname = "imapfilter";
-  version = "2.6.16";
+  version = "2.7.5";
 
   src = fetchFromGitHub {
     owner = "lefcha";
     repo = "imapfilter";
     rev = "v${version}";
-    sha256 = "0f65sg6hhv6778fxwsz4hvarbm97dsb8jj0mg7a9qs273r35pqck";
+    sha256 = "nbVwbPkNbJz4GHhvOp+QVgiBqKA/HR34p4x3NXJB7ig=";
```

Comment:
Fun: I just sent out a patch with `sha256 = "0a7f85r3axwclzw1s79zl2l8222nj2gklvvq33w9qv0dz5n71dcx"` to you (and others). Why is this a different hash? Seems to be another encoding, isn't it?

Reply:
Yes, I use `nixUnstable` so that's why I get hashes in this format. It's possible to verify I think it's the same hash, but hydra does that too anyway. Why did you send me and others that PATCH BTW?

Reply:
To update the package of course.